sncosmo/models.py

```diff
@@ -955,6 +956,70 @@ def _flux(self, phase, wave):
                 self._3d_model_flux(points).reshape(lp, lw))
 
 
+class SNEMOSource(Source):
+    """The SNEMO Type Ia supernova spectral timeseries model
+
+    The spectral flux density of this model is given by
+
+    .. math::
+       F(t, \lambda) = c_0(e_0(t, \lambda) + 
+                           \Sum_{i=1}^{n} c_i e_i(t, \lambda))
+                           \\times FM07(\lambda, A_s)
+    where ``c_0``, ``c_i``, and ``A_s`` are the free parameters of the model.
+
+    Parameters
+    ----------
+    fluxfile : str or obj, optional
+        Filename of an ascii file containing 2-d
+        array of spectral flux density values for a grid of phase
+        and wavelength values. Assuming columns ``phase``, ``wavelength``,
+        ``e_0``, ``e_1``, ``e_2``...
+    n_vector : int
+        One of 2, 7, 15 to select SNEMO2, SNEMO7, SNEMO15
```

Comment:
Update the docstring? (`n_vector` not actually an argument to the class.)

Reply:
Oops, removed this.